mitosheet/mitosheet/api/graph/get_graph.py

```diff
@@ -84,10 +94,19 @@ def get_graph(event: Dict[str, Any], steps_manager: StepsManager) -> str:
         generation_code = ""
     else:
         fig = get_plotly_express_graph(
-            graph_type, df, x_axis_column_headers, y_axis_column_headers
+            graph_type,
+            df,
+            safety_filter_turned_on_by_user,
+            x_axis_column_headers,
+            y_axis_column_headers,
         )
         generation_code = get_plotly_express_graph_code(
-            graph_type, df, df_name, x_axis_column_headers, y_axis_column_headers
+            graph_type,
+            df,
+            df_name,
+            safety_filter_turned_on_by_user,
+            x_axis_column_headers,
+            y_axis_column_headers,
```

Comment:
I think that keeping these parameters in the same order in both cases is a nice thing to do. Maybe move df_name to the end of `get_plotly_express_graph_code` - aka as the last param?